expressServer/scripts/encryptUser.js

```diff
@@ -0,0 +1,65 @@
+const mongoose = require('mongoose');
+const dotenv = require('dotenv');
+const crypto = require('crypto');
+const { create } = require('domain');
+const prompt = require('prompt-sync')();
+
+dotenv.config();
+
+mongoose.connect(process.env.MONGODB_URI, {})
+  .then(() => console.log('Connected to MongoDB'))
+  .catch(err => console.error('Could not connect to MongoDB', err));
+
+const userSchema = new mongoose.Schema({
+  username: { type: String, required: true, unique: true },
+  hash: { type: String, required: true },
```

Comment:
Required true will not allow you to insert new users using the old strategy while you are developing the new strategy